tests/js/spec/utils/profiling/gl/utils.spec.tsx

```diff
@@ -321,12 +321,6 @@ describe('Rect', () => {
 });
 
 describe('findRangeBinarySearch', () => {
-  it('throws if target is out of range', () => {
```

Comment:
We run this super often and it is a case which should be impossible. If it happens, we'll enter an infinite loop and see the error in our sentry dashboard.